### PR TITLE
added instructions for Ubuntu and Python

### PR DESCRIPTION
**Pull request by Anthonyjlem**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,42 +3,50 @@
...
 
 Vim support is likely indentical but I haven't tested it on Vim (I have a few
-neovim specific plugins for lsp, treesitter and autocompletion support)
+Neovim specific plugins for LSP, treesitter and autocompletion support)
 
-To use the fixers and linters, you should install the relevant linters + fixers
+To use the fixers and linters, you should install the relevant linters and fixers
 (see the config file for the relevant ones). You also will need to install any
-LSPs that you want.
+LSPs that you want. For Python, use pip3 to install the following: pyright,
```
Comment (kajananchinniah) [inline README.md]: Could you put a section for Python instead ? 

I feel like things would get messy if the installation for other langs are added

File: README.md [hunk 2]
```diff
@@ -3,42 +3,50 @@
...
 again for convenience.
 
-## Installation.
-Get Vim Plug (https://github.com/junegunn/vim-plug)
+## Installation
+
+First install Neovim. For Ubuntu, run the following:
+```
+sudo add-apt-repository ppa:neovim-ppa/stable
+sudo apt-get install -y neovim
```
Comment (kajananchinniah) [inline README.md:11]: Could you add a link to the Neovim installation wiki and say for other distros, refer to here

File: README.md [hunk 3]
```diff
@@ -3,42 +3,50 @@
...
+```
+
+Next, get Vim Plug (https://github.com/junegunn/vim-plug) by running the following:
 ```
 sh -c 'curl -fLo "${XDG_DATA_HOME:-$HOME/.local/share}"/nvim/site/autoload/plug.vim --create-dirs \
        https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim'
 ```
-Afterwards, simply copy and paste the init.vim file into `~/.config/nvim/init.vim`.
+Afterwards, make a `nvim` directory under `~/.config/` and copy and paste the `init.vim` file from this repository into `~/.config/nvim/init.vim`.
```
Comment (kajananchinniah) [inline README.md]: Include the method to make a directory (mkdir -p ~/.config/nvim)

**Review by kajananchinniah — CHANGES_REQUESTED** [review]
Minor changes needed, but overall the changes look good to me. Thanks for the grammar fixes!

File: README.md (hunk 1 above)
Comment (Anthonyjlem) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Anthonyjlem) [reply README.md:11]: Done

File: README.md (hunk 3 above)
Comment (Anthonyjlem) [reply README.md]: Done

**Pull request merged**